src/Tickets/Emails/JSON_LD/Order_Schema.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+namespace TEC\Tickets\Emails\JSON_LD;
+
+use TEC\Tickets\Commerce\Module;
+
+/**
+ * Class Order_Schema
+ *
+ * @since TBD
+ *
+ * @pacakge TEC\Tickets\Emails\JSON_LD
+ */
+class Order_Schema extends JSON_LD_Abstract {
+
+	/**
+	 * The type of the schema.
+	 *
+	 * @since TBD
+	 *
+	 * @var string
+	 */
+	public static string $type = 'Order';
+
+	/**
+	 * The order object.
+	 *
+	 * @since TBD
+	 *
+	 * @var \WP_Post
+	 */
+	protected \WP_Post $order;
+
+	/**
+	 * Order_Schema constructor.
+	 *
+	 * @since TBD
+	 *
+	 * @param \WP_Post $order The order object.
+	 */
+	public function __construct( \WP_Post $order ) {
+		$this->order = $order;
+	}
```

Comment:
We should avoid using the contructor, let do a factory method, like `Order_Schema::from_email( $email )`